Logger/BaseLoggerMixins.cs

```diff
@@ -1,7 +1,82 @@
-﻿namespace Logger
+﻿using System;
+
+namespace Logger
 {
     public static class BaseLoggerMixins
     {
-        
+        public static void Error(this BaseLogger baseLogger, string logLevel, params object[] message)
+        {
+            
+            if(baseLogger is null)
+            {
+                throw new ArgumentNullException();
```

Comment:
When throwing an ArgumentNullException you want to pass the name of the argument that was null.
```suggestion
                throw new ArgumentNullException(nameof(baseLogger));
```